app/components/StatusBadge.tsx

```diff
@@ -0,0 +1,68 @@
+import { Chip } from "@mui/material";
+
+interface Props {
+  label?: string;
+  variant: "complete" | "late" | "onTrack" | "inReview" | "customText" | "none";
+}
+
+const colours = {
+  complete: {
+    labelColour: "#236631",
+    badgeColour: "#EAF3EC",
+    label: "Complete",
+  },
+  late: {
+    labelColour: "#CC242A",
+    badgeColour: "#FBEAEA",
+    label: "Late",
+  },
+  onTrack: {
+    labelColour: "#FFFFFF",
+    badgeColour: "#003366",
+    label: "On track",
+  },
+  none: {
+    labelColour: "#666666",
+    badgeColour: "#E8E8E8",
+    label: "None",
+  },
+  inReview: {
+    labelColour: "#003366",
+    badgeColour: "#E6EBF0",
+    label: "In review",
+  },
+  customText: {
+    labelColour: "#666666",
+    badgeColour: "#E8E8E8",
+  },
+};
+
+const StatusBadge: React.FC<Props> = ({ label = "custom text", variant }) => {
```

Comment:
What is the custom text for?

Reply:
It's part of the [wireframe](https://www.figma.com/file/PzOY8RyQnVXnjZJdSv2AqQ/Wireframing_Figma?node-id=1301%3A2696) for that control, so I implemented it here